src/contactCard/Summary.tsx

```diff
@@ -45,7 +50,7 @@ function renderContactSummary(profile: IPersonaProfile, onContactDetailsClick: (
                 <span>{profile.officeLocation}</span>
                 <span>&nbsp;{profile.city}</span>
             </div>
-            <ActionButton className="more-details contact-details" onClick={onContactDetailsClick}>
+            <ActionButton className="more-details contact-details" onClick={handleContactDetailsClick} aria-label="Show more" aria-expanded={true} data-focus="button" aria-live="polite">
```

Comment:
Will you be able to call `handleContactDetailsClick` from this function? Is it in scope?